vendor/ghc-filesystem/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+cmake_minimum_required(VERSION 3.7.2)
+project(ghcfilesystem)
+
+include(CMakeDependentOption)
+
+cmake_dependent_option(GHC_FILESYSTEM_BUILD_TESTING
+    "Enable tests" ON
+    "CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR" OFF)
+cmake_dependent_option(GHC_FILESYSTEM_BUILD_EXAMPLES
+    "Build examples" ON
+    "CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR" OFF)
+cmake_dependent_option(GHC_FILESYSTEM_WITH_INSTALL
+    "With install target" ON
+    "CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR" OFF)
+
+if(NOT DEFINED CMAKE_CXX_STANDARD)
+    set(CMAKE_CXX_STANDARD 11)
+    set(CMAKE_CXX_STANDARD_REQUIRED ON)
+    set(CMAKE_CXX_EXTENSIONS OFF)
+endif()
+if(CMAKE_CXX_STANDARD LESS 11)
+    message(FATAL_ERROR "CMAKE_CXX_STANDARD is less than 11, ghc::filesystem only works with C++11 and above.")
+endif()
+
+add_library(ghc_filesystem INTERFACE)
```

Comment:
...here @jagerman

Reply:
Yes, I saw that, but I'm not loading that CMakeLists.txt file currently (and don't want to because it'll break compatibility with xenial because of the cmake version requirement at the top).